examples/nestjs/consumer/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "pact-nestjs-consumer",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "test:publish": "node -r ts-node/register test/pact/publish.ts",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest && npm run test:publish",
```

Comment:
Ideally, publish would only be run by CI, on distinct versions (or commits) of the source.

Reply:
Resolved in the other branch